local time = require('@module')
local utc = time.utc_now()
local now = time.now()
local la_time = now:change_zone("America/Los_Angeles")
local la_now = time.now("America/Los_Angeles")
print(`{nameof(la_now)} = {la_now}`)
print(time.current_zone)
print(la_time)
print(la_time.time_zone)
print(la_time + time.hours(100))
print(utc, utc - now)
print(now.time_zone)
print(now:format("%Z"))
print(now, now.day, now.millisecond)
local some_date = time.from_date(1999, 10, 10)
print(now.day, now.second, now.millisecond)
local t = now + time.hours(10)
print(t, t.time_zone)
print(time.now() - now)
